python/cuml/test/test_metrics.py

```diff
@@ -160,40 +163,90 @@ def test_rand_index_score(name, nrows):
     assert array_equal(cu_score, cu_score_using_sk)
 
 
-def test_mean_squared_error():
+def test_regression_metrics():
     y_true = np.arange(50, dtype=np.int)
     y_pred = y_true + 1
     assert_almost_equal(mean_squared_error(y_true, y_pred), 1.)
+    assert_almost_equal(mean_squared_log_error(y_true, y_pred),
+                        mean_squared_error(np.log(1 + y_true),
+                                           np.log(1 + y_pred)))
+    assert_almost_equal(mean_absolute_error(y_true, y_pred), 1.)
 
 
 @pytest.mark.parametrize('n_samples', [50, stress_param(500000)])
 @pytest.mark.parametrize('dtype', [np.int32, np.int64, np.float32, np.float64])
-def test_mean_squared_error_random(n_samples, dtype):
+def test_regression_metrics_random(n_samples, dtype):
     if dtype == np.float32 and n_samples == 500000:
         # stress test for float32 fails because of floating point precision
         pytest.xfail()
 
     y_true, y_pred = generate_random_labels(
         lambda rng: rng.randint(0, 1000, n_samples).astype(dtype))
+
     mse = mean_squared_error(y_true, y_pred, multioutput='raw_values')
     skl_mse = sklearn_mse(y_true, y_pred, multioutput='raw_values')
     cp.testing.assert_array_almost_equal(mse, skl_mse, decimal=2)
 
+    mae = mean_absolute_error(y_true, y_pred, multioutput='raw_values')
+    skl_mae = sklearn_mae(y_true, y_pred, multioutput='raw_values')
+    cp.testing.assert_array_almost_equal(mae, skl_mae, decimal=2)
+
+    msle = mean_squared_log_error(y_true, y_pred, multioutput='raw_values')
```

Comment:
As in other tests, it'd be better to split these tests up to be smaller and more separate.

I think you can just parameterize over "multioutput" and eliminate the duplication.

Reply:
Good point.